osu.Game/Rulesets/Mods/ModPreset.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// A mod preset is a named collection of configured mods.
+    /// Presets are presented to the user in the mod select overlay for convenience.
+    /// </summary>
+    public class ModPreset
```

Comment:
Guessing this is going to become a `RealmObject` at some point?

Reply:
That's the plan, more or less.